transit-vpc-primary-account-existing-vpc.template

```diff
@@ -80,6 +80,16 @@
       "Type" : "String",
       "Default" : ""
     },
+    "CodeLocationS3Bucket" : {
+      "Description" : "The location of the CISCO code used by this solution",
```

Comment:
Updated to be vendor agnostic labeling it generically as "transit vpc automation" code